modules/Bio/EnsEMBL/Compara/PipeConfig/ReindexMembers_conf.pm

```diff
@@ -99,12 +100,39 @@ sub default_options {
         # Main capacity for the pipeline
         'copy_capacity'                 => 4,
 
+        # Whole db DC parameters
+        'datacheck_groups' => ['compara_gene_tree_pipelines'],
+        'db_type'          => ['compara'],
+        'work_dir' => $self->o('pipeline_dir'),
```

Comment:
I'm not sure if I'd suggest vertically aligning fat commas with pre-existing code (as with the assignment of `'output_dir_path'` below), as that could make it more difficult to keep track of what's actually changed. But since you're adding these lines as a block, I think it'd be no harm to line up the fat commas here:
```suggestion
        'db_type'          => ['compara'],
        'work_dir'         => $self->o('pipeline_dir'),
```